alacritty_terminal/Cargo.toml

```diff
@@ -36,10 +35,9 @@ vte = { version = "0.11.1", default-features = false, features = ["ansi", "serde
 [target.'cfg(unix)'.dependencies]
 nix = { version = "0.26.2", default-features = false, features = ["term"] }
 signal-hook = "0.3.10"
-signal-hook-mio = { version = "0.2.1", features = ["support-v0_6"] }
 
 [target.'cfg(windows)'.dependencies]
-mio-anonymous-pipes = "0.2"
+piper = "0.2"
```

Comment:
It's this https://github.com/smol-rs/piper/ . It's maintained.

Reply:
I see a PR from two months ago with a trivial change that has seen zero action. So I have my doubts.

Reply:
@chrisduerr from dependabot....

Reply:
`piper` was extracted somewhat recently from `blocking`, which is already used in production in various places. The only reason no changes have been made is because `piper` has already been scrutinized very closely through its prior use in `blocking`. Not to mention it's not used directly in many places at the moment, so there's no issues to be filed (and therefore no changes to be made). For more context see [this](https://github.com/smol-rs/async-channel/issues/1) issue.

Reply:
> @chrisduerr from dependabot....

Dependabot doesn't make random changes unless requested. If requested and not acted upon, there's clearly *some* maintenance issues.